docs/_articles/en/user-guide/prettier.md

```diff
@@ -31,19 +28,14 @@ Using Prettier for Apex, Aura, and LWC requires some configuration.
      ]
    }
    ```
+> NOTE: The `"trailingComma": "none"` setting is required for Aura.
 
-1. If you’d like to further customize Prettier, add [other config options](https://prettier.io/docs/en/options.html).
-
-1. Install the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension for VS Code.
+After creating the local configuration file, install the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension for VS Code. If you want to ensure that all your files are formatted whenever you save them, enable the `editor.formatOnSave` setting in [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings). You can use Prettier with a pre-commit tool to reformat your files before you commit the files. For more information, see [set up a Git hook](https://prettier.io/docs/en/precommit.html).
 
-1. If you want to ensure that all your files are formatted, enable the setting `editor.formatOnSave` in VS Code. For information about configuring your settings, see [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) in the Visual Studio Code docs.
-
-1. Apex Prettier runs slower than most other formatters. In some cases formatting will not succeed because VS Code will time out the operation after 500ms. In order to ensure Apex code has enough time to format your documents we recommend changing the VS Code settings as follows.
+Prettier Apex plugin runs slower than most other formatters. In some cases, formatting will not succeed because VS Code cancels a format request after 750ms. To ensure that Apex code has enough time to format, we recommend you set the `editor.formatOnSaveTimeout` setting to configure the timeout.
```

Comment:
Updated VS code default time out from 500ms to 750ms based on
Configurable Format On Save timeout: https://code.visualstudio.com/updates/v1_22?source=post_page---------------------------
Please confirm.

Reply:
We need to remove the parts around `editor.formatOnSaveTimeout` since VSCode has deprecated the use of it in favor of just showing a message that the save operation is taking longer (https://code.visualstudio.com/updates/v1_42#_handling-slow-save-operations).

Reply:
Done.